tensorbay/client/job.py

```diff
@@ -67,6 +73,7 @@ def __init__(  # pylint: disable=too-many-arguments
         result: Optional[Dict[str, Any]],
         description: Optional[str] = "",
     ) -> None:
+        self._job_updater = job_updater
```

Comment:
since we have client in Job, can we just use client to update the job rather than use a updater?

Reply:
But in that case, we need to write the open_api_do about get_job both at class `Job` and class `JobMixin`.